e2e-tests/cypress/support/submit-and-assert-errors.js

```diff
@@ -0,0 +1,32 @@
+import { submitButton } from '../e2e/pages/shared';
+import partials from '../e2e/partials';
+
+/**
+ * @param {String} field
+ * @param {String} fieldValue - the value to input - can be null
+ * @param {Number} errorIndex - index of error in errorSummary
+ * @param {Number} errorSummaryLength - the number of expected errors in errorSummary
+ * @param {String} errorMessage
+ */
+export default (field, fieldValue, errorIndex, errorSummaryLength, errorMessage) => {
+  // if fieldValue (null if want to test empty input)
```

Comment:
Could we say "only type if a field value is provided"? Found this a bit confusing